### ISSUE TITLE
Ping times from configurable location

### ISSUE THREAD

**Issue by borekb** [nzbart/AzureSitePing#5]
It would be nice if the site collected the pings and then would be able to answer a question as "what's the ping from San Francisco to the various data centers". If enough people use it from various world-wide locations, this could be possible.

**Comment by nzbart** [comment]
Good idea! Crowd-sourcing latencies from various locations would be doable, but probably non-trivial. The difficulties I can see would be:
1. I would need to try to figure out the location of browsers using IP addresses, which isn't 100% reliable.
2. Different browsers and internet connections would have different performance characteristics, potentially skewing results.
3. Currently, this website runs purely in the user's browser - there is no server that could collate all this information and process it.

However, if enough data points were collected, these issues should be easy enough to sort out. A Google App Engine or Azure Websites back end would be ample for collecting and processing data.

The biggest problem at the moment is that I don't really have time to work on this right now. It would be an interesting project, so it could take my fancy at some point in the future, however. I'll leave it open for now as a reminder for myself and in case someone else would like to make the changes...

Thanks for the suggestion,
Bart
